docker/otel-collector/config.yaml

```diff
@@ -42,15 +42,15 @@ exporters:
     loglevel: ${env:HYPERDX_LOG_LEVEL}
   logzio/traces:
     account_token: 'X' # required but we don't use it
-    endpoint: 'http://ingestor:8002?hdx_platform=otel-traces'
+    endpoint: 'http://localhost:8002?hdx_platform=otel-traces'
```

Comment:
![logs](https://github.com/hyperdxio/hyperdx/assets/59823089/2f2d1dab-a904-4d06-84b1-b5498106da11)

The AWS container was able to communicate between containers using only localhost. Please let me know if there is a better network interface configuration!
AWS Docs, https://docs.aws.amazon.com/AmazonECS/latest/userguide/fargate-task-networking.html

Reply:
I believe this change would break the current services. I'd suggest to document these networking updates in README in `aws` dir

Reply:
The existing settings have been restored and detached to the [aws/README.md:29-58](https://github.com/hyperdxio/hyperdx/pull/79/files#diff-d1f3cfb2cd1f80df61fb16ed5304cf357114bed4c6b1a8a57a155328c5f581f7R29-R58)